app/views/lead_providers/partnership_csv_uploads/new.html.erb

```diff
@@ -0,0 +1,35 @@
+<% content_for :title, "Upload partnership csv" %>
+<% content_for :before_content, govuk_back_link( text: "Back", href: '#') %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">Upload your schools using a CSV</h1>
+    <p class="govuk-body">Instructions on how to prepare your CSV:</p>
+    <ul class="govuk-list govuk-list--bullet">
+      <li>Enter your school URN in one column</li>
+      <li>Each school URN should be placed on a new row</li>
+      <li>No other data needed</li>
+      <li>Must be under 2mb</li>
+    </ul>
+    <%= form_with model: @partnership_csv_upload, url: :lead_providers_partnership_csv_uploads do |f| %>
+      <% if !@partnership_csv_upload.errors[:base].empty? %>
+        <div class="govuk-error-summary" aria-labelledby="error-summary-title" role="alert" tabindex="-1" data-module="govuk-error-summary">
+          <h2 class="govuk-error-summary__title" id="error-summary-title">
+            There is a problem
+          </h2>
+          <div class="govuk-error-summary__body">
+            <ul class="govuk-list govuk-error-summary__list">
+            <% @partnership_csv_upload.errors[:base].each do |error| %>
+              <li>
+                <a href="#csv-error"><%= error %></a>
+              </li>
+            <% end %>
+            </ul>
+          </div>
+        </div>
+      <% end %>
```

Comment:
There is a `<%= f.govuk_error_summary %>` component that this seems to mirror.

Reply:
Not exactly, it will only display a single error at a time as there is only one input field to verify, so if we have a too big file that is not a csv, if will display only the first error in the validation method. The way it is written will display both messages at the same time.

Reply:
I think that's deliberate https://github.com/alphagov/govuk-design-system-backlog/issues/47#:~:text=if%20you%20need%20to%20tell%20the%20user%20about%20more%20than%20one%20error%20in%20a%20single%20field%2C%20show%20a%20single%20error%20message%20telling%20the%20user%20to%20do%20the%20thing%20that's%20most%20likely%20to%20fix%20the%20error

Reply:
Yes, but the errors here are not interdependent ?